frontends/mit-open/src/pages/ChannelPage/UnitChannelSkeleton.tsx

```diff
@@ -0,0 +1,204 @@
+import React, { useMemo } from "react"
+import { styled, Container, Box, Breadcrumbs, Banner } from "ol-components"
+import { MetaTags } from "ol-utilities"
+import { SearchSubscriptionToggle } from "@/page-components/SearchSubscriptionToggle/SearchSubscriptionToggle"
+import { ChannelDetails } from "@/page-components/ChannelDetails/ChannelDetails"
+import { useChannelDetail } from "api/hooks/channels"
+import ChannelMenu from "@/components/ChannelMenu/ChannelMenu"
+import ChannelAvatar from "@/components/ChannelAvatar/ChannelAvatar"
+import ResourceCarousel, {
+  ResourceCarouselProps,
+} from "@/page-components/ResourceCarousel/ResourceCarousel"
+import { SourceTypeEnum } from "api"
+import { getSearchParamMap } from "@/common/utils"
+import { HOME, UNITS } from "../../common/urls"
+import { ChannelTypeEnum } from "api/v0"
+
+const UNITS_LABEL = "MIT Units"
+
+const Page = styled.div({})
+
+export const ChannelTitleRow = styled.div`
```

Comment:
seems like we have this duplicated. it would be nice to consolidate this along with the other duplicated elements such as ChannelControls